### PR TITLE
i18n(it): Added content.mdx and talks.mdx

### PR DESCRIPTION
**Pull request by casungo**
#### Description (required)

Translated `content.mdx` and `talks.mdx` to Italian

### CODE REVIEW

File: src/content/docs/it/community-resources/talks.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+- [Chiacchierata con il caffè su tutto ciò che riguarda Astro](https://www.youtube.com/watch?v=3eyedOP83fw) con Matthew Phillips su WebStorm Webinar  
+- [Decodificare i dati degli sviluppatori: analizzare il rapporto sullo sviluppo web | Qual è il framework numero uno per il 2024?](https://youtu.be/2vqIHyz6iKs?si=vLcyzm07teC6v7u5&t=1679) con Sterling Davis e Matt Kane da Netlify  
+- [Snowpack, Astro](https://www.devtools.fm/episode/14) con Fred K. Schott su devtools.fm  
+- [Astro e MDX per il digital gardening](https://podrocket.logrocket.com/astro-mdx-kathleen-mcmahon) con Kathleen McMahon su PodRocket  
+- [Decodificare il framework web Astro](https://www.youtube.com/watch?v=4PKT8x78yOQ) con Chris Pennington su Code Ryan
+
+## Sessioni di sviluppo in live
+
+- [Costruire siti web più veloci con Astro](https://www.youtube.com/watch?v=0eka27P4Pr4) con Cassidy Williams  
+- [Astro SSR e costruire una demo di Hackernews](https://www.youtube.com/watch?v=2ZEMb_H-LYE) con Fred K. Schott e Ryan Carnito
```
Comment (delucis) [inline src/content/docs/it/community-resources/talks.mdx:46]: Typo that was fixed in #6728 

```suggestion
- [Astro SSR e costruire una demo di Hackernews](https://www.youtube.com/watch?v=2ZEMb_H-LYE) con Fred K. Schott e Ryan Carniato  
```

File: src/content/docs/it/community-resources/talks.mdx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+- [Architettura Islands | Astro Islands spiegato semplicemente](https://www.youtube.com/watch?v=kwqlP6Pe0SI) di ExcaliBear Codes
+- [Astro è un game changer per gli sviluppatori Nuxt e Vue?](https://www.youtube.com/watch?v=Sq7eKxPqfSo) di Program with Erik  
+- [Come Astro ha reso il mio sito 100 volte più veloce](https://www.youtube.com/watch?v=cOxA3kMYtkM) di James Perkins
+
+## Conferenze
+
+- [View Transitions: realtà vs finzione](https://www.youtube.com/watch?v=iT-3amHK7tA&list=PLqGQbXn_GDmkOsHI7-Wrbv1GgAA4tJZhg&index=29) di Fred Schott a ViteConf 2023
+- [Portare React nei tuoi siti di contenuti con Astro](https://www.youtube.com/watch?v=Fouws8iKPKQ) di Matthew Phillips a Future Frontend 2023  
+- [Tutto su Astro](https://www.youtube.com/watch?v=WKXHFIulZ9g) di Elian Van Cutsem, React Day Berlin 2023
+
```
Comment (delucis) [inline src/content/docs/it/community-resources/talks.mdx:33]: #7026 added an extra link here:

```md
- [You Don't Know How to SSR](https://portal.gitnation.org/contents/you-dont-know-how-to-ssr) by Emanuele Stoppa, DevOps.js Conf 2024
```

Maybe something like this?


```suggestion
- [Non sai come fare SSR](https://portal.gitnation.org/contents/you-dont-know-how-to-ssr) di Emanuele Stoppa, DevOps.js Conf 2024

```

**Review by delucis — COMMENTED** [review]
Looks great — thank you @casungo! Just noticed a couple of things that were updated in the English version since this PR was opened.

**Comment by casungo** [comment]
thx! all of the suggested changes where spot on :)

**Review by delucis — APPROVED** [review]
Thanks again for the update 🙌

**Pull request merged**